README.md

```diff
@@ -75,7 +77,7 @@ Every new feature also requires an associated sample to be added to the .NET MAU
 ### 6. Documentation Complete 
 Before a Pull Request can be merged into the .NET MAUI Toolkit, the Pull Request Author must also submit the documentation to [documentation repository](https://github.com/MicrosoftDocs/CommunityToolkit).
 
-### 7. Merged
+### 7. Completed
```

Comment:
This is now different than from what's in the chart. Isn't that confusing? Maybe also make the rest of the headings reflect the chart more?

Reply:
Ah - good catch! The [Project Column](https://github.com/CommunityToolkit/Maui/projects/1) is `Completed`, so I'll update the drawing to be `Completed` as well.

Long story short, all of the `msft-bot` automations are based around the Project Column names, so it's easier for us to change the docs to reflect the project columns than it is to change the Project Column names.